utils/kmstest.cpp

```diff
@@ -723,6 +730,7 @@ static void print_outputs(const vector<OutputInfo>& outputs)
 			for (const PropInfo &prop: p.props)
 				printf(" %s=%" PRIu64, prop.prop->name().c_str(),
 				       prop.val);
+			printf("\n");
```

Comment:
I don't think this belongs to this patch

Reply:
The universal planes and parsing order patch more or less affects a bit everything, also what makes sense in the printing phase... anyway these two patches affect each other quite a bit and keeping them separate is kind of hard. I already screwed up my first rebase fix attempt so badly, that I'll start from the scratch and try to make some sense to these.